src/dmd/target.d

```diff
@@ -310,7 +310,7 @@ struct Target
      * Returns:
      *      true if the operation is supported or type is not a vector
      */
-    extern (C++) bool isVectorOpSupported(Type type, ubyte op, Type t2 = null)
+    extern (C++) bool isVectorOpSupported(Type type, TOK op, Type t2 = null)
```

Comment:
The C++ declaration: https://github.com/dlang/dmd/blob/ac53452a31ad00dc9da1b6cab2e2c758644a6e1e/src/dmd/target.h#L91

So this won't work due to mangling differences between D `enum TOK : ubyte` and the new C++ `struct TOK`. I'd just leave it as it was.